src/Build/System.Text.Json.pkgdef

```diff
@@ -3,5 +3,5 @@
 "codeBase"="$BaseInstallDir$\MSBuild\Current\Bin\System.Text.Json.dll"
 "publicKeyToken"="cc7b13ffcd2ddd51"
 "culture"="neutral"
-"oldVersion"="0.0.0.0-4.0.1.0"
-"newVersion"="4.0.1.0"
+"oldVersion"="0.0.0.0-5.0.0.2"
+"newVersion"="5.0.0.2"
```

Comment:
This whole file should be deleted if our direction is that VS should have its own copy of this file, since MSBuild should then not be telling VS to look at the msbuild copy.

Reply:
@AArnott Do we need to coordinate an insertion of MSBuild with the other changes in the VS Repo?